webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -2442,6 +2451,14 @@ def mocked_get(url, params, **options):
         ok_('No description for this field.' in response.content)
         ok_('Description of the signature field' in response.content)
 
+        # If the user ceases to be active, these PII fields should disappear
+        User.objects.filter(id=user.id).update(is_active=False)
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+        ok_('peterbe@mozilla.com' not in response.content)
```

Comment:
You shouldn't put real email addresses in code. Why not use an ``@example.org`` fake one? That's what they're for.